usr/local/www/system_advanced_sysctl.php

```diff
@@ -135,155 +135,93 @@
-											</a>
-										</td>
-							<?php endif; ?>
-									</tr>
-								</table>
-							</td>
-						</tr>
-						<?php endforeach; unset($tunables); ?>
-						<tr>
-						<td class="list" colspan="3">
-							</td>
-							<td class="list">
-								<table border="0" cellspacing="0" cellpadding="1" summary="edit">
-									<tr>
-										<td valign="middle">
-											<a href="system_advanced_sysctl.php?act=edit">
-												<img src="./themes/<?= $g['theme']; ?>/images/icons/icon_plus.gif" width="17" height="17" border="0" alt="" />
-											</a>
-										</td>
-									</tr>
-								</table>
-							</td>
-						</tr>
-					</table>
-				</div>
-			</td>
+			<th class="col-sm-3"><?=gettext("Tunable Name"); ?></th>
+			<th><?=gettext("Description"); ?></th>
+			<th class="col-sm-1"><?=gettext("Value"); ?></th>
+			<th></th>
 		</tr>
-		<?php else: ?>
-		<tr>
-			<td>
-				<div id="mainarea">
-					<form action="system_advanced_sysctl.php" method="post" name="iform" id="iform">
-						<table class="tabcont" width="100%" border="0" cellpadding="6" cellspacing="0" summary="edit system tunable">
-							<tr>
-								<td colspan="2" valign="top" class="listtopic"><?=gettext("Edit system tunable"); ?></td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top" class="vncellreq"><?=gettext("Tunable"); ?></td>
-								<td width="78%" class="vtable">
-									<input size="65" name="tunable" value="<?php echo $pconfig['tunable']; ?>" />
-								</td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top" class="vncellreq"><?=gettext("Description"); ?></td>
-								<td width="78%" class="vtable">
-									<textarea rows="7" cols="50" name="descr"><?php echo $pconfig['descr']; ?></textarea>
-								</td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top" class="vncellreq"><?=gettext("Value"); ?></td>
-								<td width="78%" class="vtable">
-									<input size="65" name="value" value="<?php echo $pconfig['value']; ?>" />
-								</td>
-							</tr>
-							<tr>
-								<td width="22%" valign="top">&nbsp;</td>
-								<td width="78%">
-									<input id="submit" name="Submit" type="submit" class="formbtn" value="<?=gettext("Save"); ?>" />
-									<input type="button" class="formbtn" value="<?=gettext("Cancel");?>" onclick="window.location.href='<?=$referer;?>'" />
-									<?php if (isset($id) && $a_tunable[$id]): ?>
-									<input name="id" type="hidden" value="<?=htmlspecialchars($id);?>" />
-									<?php endif; ?>
-								</td>
-							</tr>
-						</table>
-					</form>
-				</div>
-			</td>
-		</tr>
-		<?php endif; ?>
-	</table>
-<?php include("fend.inc"); ?>
-</body>
-</html>
+	</thead>
+	<?php foreach ($tunables as $i => $tunable):
+		if (!isset($tunable['modified']))
+			$i = $tunable['tunable'];
+	?>
+	<tr>
+		<td ondblclick="document.location='system_advanced_sysctl.php?act=edit&amp;id=<?=$i;?>';">
```

Comment:
The ondblclick attribute should be removed; if you want to keep this functionality, can you move it to a ```<script>events.push(function{})</script>``` section? It'll automatically be executed onload